modules/@apostrophecms/asset/index.js

```diff
@@ -753,32 +757,28 @@ module.exports = {
               const jsFilename = JSON.stringify(component);
               const name = getComponentName(component, options, i);
               const jsName = JSON.stringify(name);
+              const importName = `${name}${options.importSuffix || ''}`;
```

Comment:
Suffix was not used consistently, could have caused a bug, I refactored to make that easier.